insta_scrape.py

```diff
@@ -0,0 +1,70 @@
+from selenium.webdriver.common.by import By
+import time
+
+waiting_seconds = 4
+
+
+def handle_cookiebanner(driver, allow_cookies=1):
+    print('handle cookie banner..')
```

Comment:
kann weg?

Reply:
No, Logging is imporant. While using linux system we can use bash scripts to log our executions.

```pyhton
$ python main.py <username> <password> > /var/log/some_log.log
```

If you dont want any output on Commandline use "> /dev/null 2>&1".

Reply:
But yeah, these print lines are very minimalistic and have to be refactored at any point.